jetty-ant/src/main/java/org/eclipse/jetty/ant/ServerProxyImpl.java

```diff
@@ -422,8 +420,7 @@ private void configureHandlers()
             {
                 handlers = new HandlerCollection();
                 server.setHandler(handlers);
-                handlers.setHandlers(new Handler[] { contexts, new DefaultHandler(),
-                        requestLogHandler });
+                handlers.setHandlers(new Handler[] { contexts, new DefaultHandler() });
```

Comment:
Ditto, do we support `RequestLogHandler` at the end of the handler list still?

Reply:
ditto.  We do support it as an alternative in a handler list, but it should be the first not the last.  This is tested in RequestLogTest